docs/UserGuide.md

```diff
@@ -16,23 +16,22 @@ and you can type fast, UniGenda can get your contact management tasks done faste
    4. [Editing a person](#editing-a-person--edit)
    5. [Locating persons by name](#locating-persons-by-name-find)
    6. [Deleting a person](#deleting-a-person--delete)
-   7. [Viewing a person's schedule](#viewing-a-persons-schedule-viewschedule)
-   8. [Viewing contacts by tags](#viewing-contacts-by-tags-viewgroup)
-   9. [Adding a person's schedule](#adding-a-persons-schedule-addevent)
-   10. [Editing a person's schedule](#editing-a-persons-schedule-editevent)
-   11. [Deleting a person's schedule](#deleting-a-persons-schedule-deleteevent)
-   12. [Getting persons who are free](#getting-persons-who-are-free-freeschedule)
-   13. [Importing a person's schedule](#importing-a-persons-schedule-import)
-   14. [Exporting a person's schedule](#exporting-a-persons-schedule-export)
-   15. [Clearing all entries](#clearing-all-entries--clear)
-   16. [Exiting the program](#exiting-the-program--exit)
-   17. [Saving the data](#saving-the-data)
-   18. [Editing the data file](#editing-the-data-file)
-3. [Coming Soon](#coming-soon-v13)
-   1. [Viewing Schedule](#viewing-a-persons-schedule-viewschedule)
-   2. [Getting common free timing of persons by tag](#getting-common-free-timing-of-persons-by-tag-findcommontiming)
-4. [FAQ](#faq)
-5. [Command Summary](#command-summary)
+   7. [Setting a contact as the user](#setting-a-contact-as-the-user--setuser)
+   8. [Viewing Schedule](#viewing-a-persons-schedule-viewschedule)
+   9. [Viewing contacts by tags](#viewing-contacts-by-tags-viewgroup)
+   10. [Adding a person's schedule](#adding-a-persons-schedule-addevent)
+   11. [Editing a person's schedule](#editing-a-persons-schedule-editevent)
+   12. [Deleting a person's schedule](#deleting-a-persons-schedule-deleteevent)
+   13. [Getting persons who are free](#getting-persons-who-are-free-freeschedule)
+   14. [Getting common free timing of persons by tag](#getting-common-free-timing-of-persons-by-tag-findcommontiming)
+   15. [Importing a person's schedule](#importing-a-persons-schedule-import)
+   16. [Exporting a person's schedule](#exporting-a-persons-schedule-export)
```

Comment:
These here are wrong. Maybe you can try to merge the upstream master with your branch first?
```suggestion
   14. [Getting common free timing of persons by tag](#getting-common-free-timing-of-persons-by-tag-whoisfree)
   15. [Importing a person's schedule](#importing-a-persons-schedule-importschedule)
   16. [Exporting a person's schedule](#exporting-a-persons-schedule-exportschedule)
```